### ISSUE TITLE
[FEATURE REQUEST] Be able to edit our submitted reviews

### ISSUE THREAD

**Issue by francisco-camargo** [oms-fyi/reviews#40]
Hello, I would love to again be able to edit reviews I have submitted. Maybe I won't be able to ever edit reviews made prior to the big change but going forward it would be great to have an edit button. Sometimes I want to fix small grammatical mistakes, but sometimes how I feel about a course changes once I have some distance from the semester and I like to be able to go back and add such an addendum.

Thanks! The new site is looking great!

**Comment by m4ttsch** [comment]
Yeah, I hear you. Originally, I wanted to keep the submission process as simple as possible, because once you get into full CRUD mode, the one-time code mechanism gets a bit unwieldy IMO. I think this is an important feature to have, and it's going to require me to put a bit more thought into how I want to handle session management, so folks aren't emailing themselves Twilio codes ten times a day.
